embedded-graphics/src/primitives/circle.rs

```diff
@@ -272,6 +271,23 @@ mod tests {
     use super::*;
     use crate::pixelcolor::BinaryColor;
 
+    #[test]
+    fn issue_143_stroke_and_fill() {
+        let circle_no_stroke: Circle<BinaryColor> =
+            Circle::new(Point::new(10, 16), 3).fill(Some(BinaryColor::On));
+        let circle_stroke: Circle<BinaryColor> = Circle::new(Point::new(10, 16), 3)
+            .fill(Some(BinaryColor::On))
+            .stroke(Some(BinaryColor::On));
+
+        assert_eq!(circle_stroke.size(), circle_no_stroke.size());
+        assert_eq!(
```

Comment:
This assert is redundant, because the iters are checked for equality below.

Reply:
Ah yeah, it's just noise at this point. Removed in b875b8c.